src/Symfony/Component/Message/Transport/ReceiverInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Message\Transport;
+
+/**
+ * @author Samuel Roze <samuel.roze@gmail.com>
+ */
+interface ReceiverInterface
+{
+    public function receive(): \Generator;
```

Comment:
`iterable`?

Reply:
This has been done then reverted to `\Generator` for the reasons described in [this comment](https://github.com/symfony/symfony/pull/24411#issuecomment-372594323).